content/doc/book/pipeline/getting-started.adoc

```diff
@@ -49,7 +49,7 @@ A basic Pipeline can be created in either of the following ways:
   control repository.
 
 The syntax for defining a Pipeline with either approach is the same, but while
-Jenkins supports entering Pipeline Script directly into the web UI, it's
+Jenkins supports entering Pipeline directly into the web UI, it's
```

Comment:
My inclination is to keep it singular since one only adds a single Pipeline into the web UI at a time.